src/ResourceManagement/SqlManagement/Microsoft.Azure.Management.Sql/Generated/DatabasesOperationsExtensions.cs

```diff
@@ -0,0 +1,1149 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
@nathannfan You should do this either by using your fork, or by creating a new branch in the sdk for people to work from.  Work that is not intended for release shoudl not go in the AutoRest branch.